.stylelintrc

```diff
@@ -13,5 +13,6 @@
     "number-leading-zero": null,
     "selector-type-case": null,
     "selector-type-no-unknown": null
-  }
+  },
+  ignoreFiles: ["packages/**/*.spec.{js,ts,tsx}"]
```

Comment:
Test files are ignored in the [yarn lint:css](https://github.com/zendeskgarden/react-components/blob/main/package.json#L17) command, but not ignored in the [.lintstagedrc](https://github.com/zendeskgarden/react-components/blob/main/.lintstagedrc#L3) file. Because I ran into some wild stylelint issues when writing tests using css modifiers, I added this line to ignore `stylelint` when husky commands are ran.

Alternatively, I also tried modifying `.lintstaged` file with something like `stylelint --ignore-files "packages/**/*.spec.{js,ts,tsx}"`, but had no luck. Open to suggestions for getting around css modifier linting issues.

Reply:
Seems fine. I think we tend to run `git commit --no-verify` in order to get around issues like this.

Reply:
That sounds good. I ended up leaving this since `--no-verify` would skip the eslint, jest, and prettier too. But I can manually run those commands and use `--no-verify` if that is preferred 👍 .

Reply:
I think this makes sense. We `stylelint` is for production styling and CSS. Tests should be fine without it.